refinery/core/views.py

```diff
@@ -715,13 +715,18 @@ def solr_core_search(request):
 def solr_select(request, core):
     # core format is <name_of_core>
     # query.GET is a querydict containing all parts of the query
-    # TODO: handle runtime errors when making GET request
-
     url = settings.REFINERY_SOLR_BASE_URL + core + "/select"
     data = request.GET.urlencode()
     try:
         full_response = requests.get(url, params=data)
-        full_response.raise_for_status()
+        # Solr sends back an additional 400 here in the data_sets 1 filebrowser
```

Comment:
API will stick around. It's used as a UI common services solr.js, which is used by multiple features.